### ISSUE TITLE
Mass Update User Timezones

### ISSUE THREAD

**Issue by CamoMacdonald** [salesagility/SuiteCRM#3216]
#### Issue

When creating multiple(sometimes hundreds) users in several time zones, it can sometimes be problematic if those users forget to define they’re own timezones on setup. It would be good if there was a mass update option from list view to set their timezones.

#### Context

Suggestion

#### Your Environment

* SuiteCRM Version used: 7.8.2
* Operating System and version (e.g Ubuntu 16.04): Ubuntu 14.04

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Mass Update User Timezones](https://trello.com/c/VQS74Tw8/19-mass-update-user-timezones)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
